tests/ffi/dev-mem_not_present/test.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -euvx
+
+# shellcheck disable=SC1091
+. ../../e2e/lib/utils
+
+check_dev-mem_not_present(){
+    expected_result="ls: cannot access '/dev/mem': No such file or directory"
+    return_from_cmd=$(podman exec -t qm bash -c "ls -l /dev/mem")
```

Comment:
$(podman exec..). probably should be enough..

Reply:
There is no need for ls. Just use `test` or `[`. podman forwards the return value.

```
if podman exec -t qm test -f /dev/mem &>/dev/null; then
...
fi
```

Reply:
Maybe rather use ` test -e` because it's a character special file

Reply:
@dougsland @dennisbrendel Thank you both for the review. 
I need to ask the design approver whether it would be better to use` test -e` instead of `ls` command.

Reply:
Done
@dennisbrendel thank you so much for the suggestion.